portal-api/src/v1/gef/controllers/application-submit.js

```diff
@@ -84,6 +88,64 @@ const addUkefFacilityIdToFacilities = async (applicationId) => {
   return facilities;
 };
 
+// retrieves user information from database
+const getUserInfo = async (userId) => {
+  const userCollectionName = 'users';
+
+  const userCollection = await db.getCollection(userCollectionName);
+  const {
+    firstname: firstName = '',
```

Comment:
do we need to lowercase - noticed in a another file it's using the lowercase property to create camelCase property value?

Reply:
this will be a big job to fix maybe? it is stored as firstname for some reason.

Reply:
done